### PR TITLE
Add RefreshAuthority And UnverifiedTransactions Methods 

### LINKED ISSUE #1: Add RefreshAuthority And UnverifiedTransactions Methods
Add `UnverifiedTransactions` method to get unverified transactions
And
Add `RefreshAuthority` method to update your authority expiration time.

### PR DESCRIPTION
**Pull request by Mahmoud-Sagharjoughi**
Resolve #1

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -153,6 +184,74 @@ func (zarinpal *Zarinpal) PaymentVerification(amount int, authority string) (ver
...
 
+// UnverifiedTransactions gets unverified transactions.
+//
+// If error is not nil, you can check statusCode for
+// specific error handling based on Zarinpal error codes.
+// If statusCode is not 100, it means Zarinpal raised an error
+// on their end and you can check the error code and its reason
+// based on their documentation placed in
+// https://github.com/ZarinPal-Lab/Documentation-PaymentGateway/archive/master.zip
+func (zarinpal *Zarinpal) UnverifiedTransactions() (statusCode int, authorities []UnverifiedAuthority, err error) {
```
Comment (sinabakh) [inline main.go]: @Mahmoud-Sagharjoughi Its better to return authorities as a first return argument.

**Review by sinabakh — CHANGES_REQUESTED** [review]

File: main.go (hunk 1 above)
Comment (Mahmoud-Sagharjoughi) [reply main.go]: @sinabakh It Changed.

File: example_test.go [hunk 2]
```diff
@@ -47,6 +48,33 @@ func ExampleZarinpal_PaymentVerification() {
...
+	}
+	log.Println("verified:", verified, "refID:", refID)
+}
+
+func ExampleZarinpal_UnverifiedTransactions() {
+	zarinPay, err := zarinpal.NewZarinpal("XXXX-XXXX-XXXX-XXXX", true)
+	if err != nil {
 		log.Fatal(err)
 	}
+	statusCode, authorities, err := zarinPay.UnverifiedTransactions()
```
Comment (sinabakh) [inline example_test.go]: @Mahmoud-Sagharjoughi Please fix here too.

Comment (Mahmoud-Sagharjoughi) [reply example_test.go]: @sinabakh Oh :/.
It fixed too.

**Review by sinabakh — APPROVED** [review]

**Comment by sinabakh** [comment]
Everything seems OK, thanks @Mahmoud-Sagharjoughi

**Pull request merged**